app/public/receiver.js

```diff
@@ -10,6 +10,16 @@ const getCurrentRoom = () => {
     });
 }
 
+const switchHumanTool = () => {
+    WS.send({
+      action: {
+        name: "switchHumanTool",
+      },
+      username: USERNAME_COOKIE,
+      gameId: GAME_ID_COOKIE,
+    });
+  };
```

Comment:
This function is out of place. We don't want to send WS data in `reciever.js`.
The actual `switchHumanTool()` is rightfully in `human.js`. This copy is unused and can be deleted